pkg/rpc/server/server.go

```diff
@@ -66,6 +67,7 @@ type (
 		log              *zap.Logger
 		https            *http.Server
 		shutdown         chan struct{}
+		started          *atomic.Bool
```

Comment:
Why is it a pointer?

Reply:
It doesn't matter much, but historically we return `Server` from `New` and not `*Server`.